src/stores/AccountsStore.js

```diff
@@ -117,7 +116,7 @@ export default class AccountsStore extends Container<AccountsState> {
     });
   }
 
-  async loadAccountTxs() {}
+  async loadAccountTxs() { }
```

Comment:
The rule is: one feature (logical change) - one PR. Make a separate PR for these changes, please.

Reply:
That's a linter cleanup, I'm ok with other rules, I guess we can accept that right?